2024-01-02 16:17:37,845 INFO    MainThread:2127232 [wandb_setup.py:_flush():76] Current SDK version is 0.16.1
2024-01-02 16:17:37,845 INFO    MainThread:2127232 [wandb_setup.py:_flush():76] Configure stats pid to 2127232
2024-01-02 16:17:37,845 INFO    MainThread:2127232 [wandb_setup.py:_flush():76] Loading settings from /home/nlp-04/.config/wandb/settings
2024-01-02 16:17:37,850 INFO    MainThread:2127232 [wandb_setup.py:_flush():76] Loading settings from /data/2_data_server/nlp-04/lost_technology/wandb/settings
2024-01-02 16:17:37,850 INFO    MainThread:2127232 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2024-01-02 16:17:37,850 INFO    MainThread:2127232 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2024-01-02 16:17:37,850 INFO    MainThread:2127232 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program_relpath': 'test.py', 'program_abspath': '/data/2_data_server/nlp-04/lost_technology/test.py', 'program': '/data/2_data_server/nlp-04/lost_technology/test.py'}
2024-01-02 16:17:37,851 INFO    MainThread:2127232 [wandb_init.py:_log_setup():524] Logging user logs to /data/2_data_server/nlp-04/lost_technology/wandb/run-20240102_161737-gj8ivyp3/logs/debug.log
2024-01-02 16:17:37,851 INFO    MainThread:2127232 [wandb_init.py:_log_setup():525] Logging internal logs to /data/2_data_server/nlp-04/lost_technology/wandb/run-20240102_161737-gj8ivyp3/logs/debug-internal.log
2024-01-02 16:17:37,852 INFO    MainThread:2127232 [wandb_init.py:init():564] calling init triggers
2024-01-02 16:17:37,852 INFO    MainThread:2127232 [wandb_init.py:init():571] wandb.init called with sweep_config: {}
config: {}
2024-01-02 16:17:37,852 INFO    MainThread:2127232 [wandb_init.py:init():614] starting backend
2024-01-02 16:17:37,852 INFO    MainThread:2127232 [wandb_init.py:init():618] setting up manager
2024-01-02 16:17:37,854 INFO    MainThread:2127232 [backend.py:_multiprocessing_setup():105] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2024-01-02 16:17:37,855 INFO    MainThread:2127232 [wandb_init.py:init():624] backend started and connected
2024-01-02 16:17:37,857 INFO    MainThread:2127232 [wandb_init.py:init():716] updated telemetry
2024-01-02 16:17:37,864 INFO    MainThread:2127232 [wandb_init.py:init():749] communicating run to backend with 90.0 second timeout
2024-01-02 16:17:38,433 INFO    MainThread:2127232 [wandb_run.py:_on_init():2254] communicating current version
2024-01-02 16:17:38,565 INFO    MainThread:2127232 [wandb_run.py:_on_init():2263] got version response 
2024-01-02 16:17:38,565 INFO    MainThread:2127232 [wandb_init.py:init():800] starting run threads in backend
2024-01-02 16:17:40,687 INFO    MainThread:2127232 [wandb_run.py:_console_start():2233] atexit reg
2024-01-02 16:17:40,687 INFO    MainThread:2127232 [wandb_run.py:_redirect():2088] redirect: wrap_raw
2024-01-02 16:17:40,692 INFO    MainThread:2127232 [wandb_run.py:_redirect():2153] Wrapping output streams.
2024-01-02 16:17:40,692 INFO    MainThread:2127232 [wandb_run.py:_redirect():2178] Redirects installed.
2024-01-02 16:17:40,692 INFO    MainThread:2127232 [wandb_init.py:init():841] run started, returning control to user process
2024-01-02 16:17:46,900 WARNING MsgRouterThr:2127232 [router.py:message_loop():77] message_loop has been closed
